typescript/looker/rtl/extensionSdk.ts

```diff
@@ -22,14 +22,16 @@
  * THE SOFTWARE.
  */
 
-import { DefaultSettings } from './apiSettings'
+import { DefaultSettings, IApiSettings } from './apiSettings'
 import { Looker31SDK } from '../sdk/3.1/methods'
 import { ExtensionSession } from './extensionSession'
 import { ExtensionTransport } from './extensionTransport'
 import {
   Authenticator,
   ITransportSettings,
 } from './transport'
+import { APIMethods } from "./apiMethods";
+import { IAuthSession } from "./authSession";
```

Comment:
Surprised prettier didn't transform those to single quote, no semis

Reply:
you probably have your IDE configured to run it automatically.

Reply:
Those were auto-added by Intellij